alpha/alpha-core/src/main/java/org/apache/servicecomb/pack/alpha/core/fsm/event/internal/ComponsitedCheckEvent.java

```diff
@@ -17,10 +17,17 @@
 
 package org.apache.servicecomb.pack.alpha.core.fsm.event.internal;
 
+import org.apache.servicecomb.pack.alpha.core.fsm.TxState;
 import org.apache.servicecomb.pack.alpha.core.fsm.event.base.TxEvent;
 
 public class ComponsitedCheckEvent extends TxEvent {
 
+  private TxState preState;
+
+  public TxState getPreState() {
```

Comment:
This states is used store the state of componsited command execution state. 
It's better to update the name to reflect the meaning of this variable.

Reply:
I renamed that `preState` to `preComponsitedState`